modality-probe-cli/src/visualize/graph.rs

```diff
@@ -219,13 +219,26 @@ fn graph_to_tree<'a>(
         });
 
         if let Ok(emeta) = meta::get_event_meta(cfg, &node.probe_id, &node.id) {
-            let payload =
-                meta::parsed_payload(emeta.type_hint.as_ref().map(|s| s.as_ref()), node.payload)
-                    .unwrap_or(None);
+            let payload = node.payload.and_then(|pl| {
+                emeta.type_hint.as_ref().and_then(|th| {
+                    meta::parsed_payload(Some(th.as_ref()), Some(pl))
+                        .ok()
+                        .flatten()
+                })
+            });
+            let has_log_str = emeta.description.contains("{}");
```

Comment:
The `DescriptionFormat` trait has a `contains_formatting()` method for this.
Should make it easier for us to expand later on for extra formatting, like `{:X}` for hex.